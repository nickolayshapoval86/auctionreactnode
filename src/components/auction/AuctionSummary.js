import React from 'react';

const AuctionsSummary = ({auction}) => {
  return (
    <div className="card z-depth-0 auction-summary">
      <div className="card-content grey-text text-darken-3">
        <span className="card-title">{auction.title}</span>
        <p>Post by the {auction.creator.name}</p>
        <p className="grey-text">{auction.description}</p>
      </div>
    </div>
  );
}

export default AuctionsSummary;