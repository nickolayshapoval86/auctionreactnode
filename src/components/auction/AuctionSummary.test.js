import { shallow } from 'enzyme';
import React from 'react';
import AuctionSummary from './AuctionSummary';

test('AuctionSummary snapshot', () => {
    const mockAuction = {
        title:'Test auction title',
        description: 'Auction description text',
        creator: {
            name: 'Auction creator'
        }
    };
    expect(shallow(<AuctionSummary auction={mockAuction} />)).toMatchSnapshot();
});