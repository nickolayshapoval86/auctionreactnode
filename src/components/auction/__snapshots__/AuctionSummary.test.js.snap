// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuctionSummary snapshot 1`] = `ShallowWrapper {}`;
